pkgs/applications/science/electronics/kicad/default.nix

```diff
@@ -197,13 +200,16 @@ stdenv.mkDerivation rec {
     "--prefix XDG_DATA_DIRS : ${cups}/share"
     "--prefix GIO_EXTRA_MODULES : ${gnome3.dconf}/lib/gio/modules"
 
-    "--set KISYSMOD ${footprints}/share/kicad/modules"
```

Comment:
You could get away with a rebuild if you'd use the same order of `makeWrapperArgs` for the default build configuration.

Also, it would be better to set the value up above under `let`, and use it here.